### ISSUE TITLE
VoiceOver does not read new messages

### ISSUE THREAD

**Issue by niccodes** [tommyxchow/frosty#468]
I was looking for a straightforward way to have Twitch chat read out loud to me while driving on a road trip. I thought an easy way to do this would be to use the VoiceOver screen reading functionality in iOS.

When using this feature with the Twitch app, each new chat message is read out, but lots of unnecessary information is prepended and appended to each chat message (details of chatter badges, instructions on how to reply). The same is true of the Twitch web chat popout.

I thought this could be solved by turning to a 3rd party app that might actually care a bit more about how the chat is presented. Unfortunately, Frosty seems to be missing an important piece to work well with VoiceOver. While regular system-provided VoiceOver facilities do function, the app appears to fail to inform VoiceOVer of new chat messages, and so does not read them out unless directed to by sliding / tapping on the screen.

I can't say I'm an experienced user of VoiceOver, so could very well be doing something wrong but it seems like Frosty could be improved for screen reader users by better informing the system accessibility framework when new chat messages are received.
